scripts/reborn_webui_v2_live_qa/test_refresh_google_oauth.py

```
@@ -0,0 +1,207 @@
+            io.BytesIO(b"not-json"),
+        )
+        with patch.dict(os.environ, self.env, clear=True), patch.object(
+            refresh_google_oauth.urllib.request, "urlopen", side_effect=error
+        ):
+            token, status = refresh_google_oauth.refresh_access_token()
+        self.assertIsNone(token)
+        self.assertEqual(status, "http_503")
+
+    def test_unreadable_http_error_body_is_sanitized(self):
+        class UnreadableBody:
+            def read(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `read`

Add return type annotation: `NoReturn`

(ANN202)